modules/aggs-matrix-stats/src/main/java/org/elasticsearch/search/aggregations/support/ArrayValuesSourceAggregationBuilder.java

```diff
@@ -29,12 +29,7 @@
 import org.elasticsearch.search.aggregations.AggregatorFactory;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
```

Comment:
Wildcard imports are forbidden in our style guide.  Not sure why that isn't being enforced automatically right now (I'm looking into it), but even so, please leave the imports as is.

See https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#java-language-formatting-guidelines. I recommend you follow the instructions there to set up your IDE to use our auto-formatting settings.

Reply:
Hi Mark, intellij automatically applied wildcard imports, sorry about that.  
I just followed the instructions and configured my IDE.